cfgov/core/utils.py

```diff
@@ -24,18 +24,26 @@
 
 LINK_ICON_TEXT_CLASSES = ['a-link_text']
 
-# Regular expression to match <a> links in HTML strings
-A_TAG = re.compile(
-    # Match an <a containing any attributes
-    r'<a [^>]*?>'
-    # And match everything inside before the closing </a>
-    r'.+?(?=</a>)'
-    # Then match the closing </a>
-    r'</a>'
+# Regular expression format string that will match any tag <tag_name> and group
+# its contents.
+TAG_RE = (
+    # Match an <tag_name[ attributes]>. If tag_name is not followed by a space
+    # and any characters except >, it must be followed by >.
+    r'<{tag_name}(?:\s[^>]+?|)>'
```

Comment:
This will not work because it could match `<aside`. The intention here is to create a non-capturing group, `(?:`, that matches any space, `\s`, followed by one or more (non-greedy) character that is *not*` `>`, `[^>]+?`, *or* match nothing after the tag name,`|)`.

Reply:
Addressed above.